src/theory/uf/cardinality_extension.cpp

```diff
@@ -1056,7 +1049,8 @@ void SortModel::addCliqueLemma(std::vector<Node>& clique)
 {
   Assert(d_hasCard);
   Assert(d_cardinality > 0);
-  while( clique.size()>size_t(d_cardinality+1) ){
+  while (clique.size() > d_cardinality + 1)
```

Comment:
This is were you compare against `size_t`. You can leave it as it is, though (this one is safe).